web/styles/left_sidebar.css

```diff
@@ -111,7 +111,7 @@
     font-weight: normal;
 
     .topic_search_section {
-        margin: 3px 0 3px var(--left-sidebar-toggle-width-offset);
+        margin: 3px 0;
```

Comment:
This and other changes for the topic search section seem to result in its width not being right. It's small enough I don't think anyone will notice in the next week. So I think it probably still makes sense to merge this and just try to fix forward before this goes to Cloud, given the next step is to finish redoing that component.